### ISSUE TITLE
iOS Headers not compatible with RN 0.40+

### ISSUE THREAD

**Issue by paulwcy** [airbnb/lottie-react-native#12]
Tried installing in my existing (and also new) project and I'm getting
```
'RCTView.h' not found
'RCTViewManager.h' not found
```

**Comment by grifotv** [comment]
Getting the same here:

```
In file included from /Users/username/reponame/node_modules/lottie-react-native/lib/ios/LottieReactNative/LottieAnimationViewManager.m:9:
/Users/username/reponame/node_modules/lottie-react-native/lib/ios/LottieReactNative/LottieAnimationViewManager.h:9:9: fatal error: 'RCTViewManager.h' file not found
#import "RCTViewManager.h"
        ^
1 error generated.
```

**Comment by lelandrichardson** [comment]
I'll push out an update that's compatible with RN 0.40. Sorry about that!

**Comment by lelandrichardson** [comment]
Fixed by #28
